### DISCUSSION TITLE
[FEATURE REQ] Make torznab feed category information usable in filters

### DISCUSSION THREAD

**Discussion by volvic2** [autobrr/autobrr#1694] [Ideas]
At the moment autobrr doesn't seem to use the category information of torznab feeds.
For example I got a torznab feed containing these elements for a torrent:
`<torznab:attr name="category" value="2040"/>
<torznab:attr name="category" value="101049"/>`
The filter doesn't seem recognize the categories at all. Neither when I set a category filter to 2040 nor when I set the filter to "Movies/HD". 
In the release tab of autobrr I also see no category information in the release's details:
`Details
Release: *releasename*
Indexer: *indexername*
Protocol: torrent
Size: 19,8 GB
Title: *title*
Year: 2024
Resolution: 2160p
Source: WEB
Codec: H.265
Group: *groupname*

So I'd like to ask, if I'm just thinking wrong or if this feature is currently missing. If it's missing, I'd like to open a request for making the torznab feed category information usable in filters .

**Comment by zze0s** [comment]
Hi! What version are you on and what does your filter look like?

The filters for category from Torznab feeds should work well, according to the tests. It's just not shown in the details box. Set log level to Trace in settings -> logs and it should show under `Categories` on the release which it has. And on debug level you should see rejections when it doesn't match.

**Comment by zze0s** [comment]
This is already implemented and we recently added support for the feeds to set categories to search. https://github.com/autobrr/autobrr/releases/tag/v1.72.0
